P/PROJ/build_tarballs.jl

```diff
@@ -90,8 +87,8 @@ dependencies = [
     # Host SQLite needed to build proj.db
     HostBuildDependency("SQLite_jll")
     Dependency("SQLite_jll")
-    Dependency("Libtiff_jll")
-    Dependency("LibCURL_jll"; compat="7.73")
+    Dependency("Libtiff_jll"; compat="4.5")
```

Comment:
```suggestion
    Dependency("Libtiff_jll"; compat="4.5.1")
```
Apparently needed since 4.5.0 doesn't exist.